.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ commands:
     parameters:
       ats_version:
         type: string
-        default: 0.1.4
+        default: 0.10.2
```

Comment:
this will get outdated quickly, can't we just unpin the version and use whatever Architect has defined?

Reply:
Not when the command is templated out like this, apparently. Using the `architect/ats` step directly would work (I think) but somehow the defaulting that happens there is dropped when doing it this way. I wasn't quite sure what the future of CI was going to look like on this repo so just wanted to get it working, but if ats stays in the mix then I'd align this with the other repos and remove the intermediate `run-ats` thing

Reply:
Makes sense. I guess this is just too old, we can migrate it later